### PR TITLE
Introducing Station.MIKE

### PR DESCRIPTION
**Pull request by madumalt**

### CODE REVIEW

File: curwmysqladapter/station.py [hunk 1]
```diff
@@ -42,6 +43,7 @@ class Station(Enum):
 
         WRF: 100000,
         FLO2D: 100000,
+        MIKE: 1300000,
```
Comment (gihankarunarathne) [inline curwmysqladapter/station.py]: @madumalt  This's the range of values that's going to use. So, it should be
```
MIKE: 100000,
```

File: curwmysqladapter/station.py [hunk 2]
```diff
@@ -77,6 +80,7 @@ def getType(name):
 
             'WRF': Station.WRF,
             'FLO2D': Station.FLO2D,
+            'MIKE': Station.MIKE,
```
Comment (gihankarunarathne) [inline curwmysqladapter/station.py]: 👍

File: curwmysqladapter/station.py (hunk 1 above)
Comment (madumalt) [reply curwmysqladapter/station.py]: corrected it

**Comment by gihankarunarathne** [comment]
👍

**Pull request merged**